noisyatom/blogs/templates/list.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'background-page.html' %}
+
+{% load bootstrap4 %}
+{% load urlify %}
+
+{% block content %}
+
+<div class="row">
+
+	<div class="col-12">
+		<div class="col-md-8">
+			<form action="" method="GET" class="form-inline">
+				<div class="col-md-8">
+					<div class="form-group">
+						<input type="text" class="col-md-6 form-control" name="q" placeholder="Search ..."
+						 value="{{request.GET.q}}">
+						
+						<button type="submit" class="btn btn-success"><i class="fa fa-search" aria-hidden="true"></i></button>
+						
+					</div>
+				</div>
+			</form>
+		</div>
+	</div>
+	<br>
+	
+	<div class="col-12">
+		<div class="col-md-2 col-md-offset-10">
+		{% if user.is_authenticated and user.is_staff %}
+			<a href="{% url 'blogs:create' %}" title="">
+			<button type="button" class="btn btn-success pull-right">Add</button>
+			</a>
+		{% endif %}
+		</div>
+	</div>
+</div>
+
+<div class="row">
+	{% for obj in object_list %}
+	<div class="col-md-12"> 
+		<div class="col-md-8 text-justify post-detail-item">
+			<h3>
+				<a href="{% url 'blogs:detail' slug=obj.slug %}">{{obj.title|upper}}</a>
+				<small>{{obj.publish}}</small>
+			</h3>
+			
+			<div class="">{{obj.get_markdown|truncatechars_html:200}}</div>
+
+			{% if instance.draft %}
+				<b>Draft:</b> <i>{{instance.publish}}</i>
+			{% else %}
+				<i>{{instance.publish}}</i>
+			{% endif %}
+			
+			<a href="{% url 'blogs:detail' slug=obj.slug %}" class="">
+				<button type="button" class="btn btn-primary pull-right">More ...</button>
+			</a>	
+		</div>
+
+		<div class="col-md-4">
+				<br><br>
+				{% if obj.image %}
+					<img src="{{obj.image.url}}" class="img-thumbnail img-responsive" />
```

Comment:
Are both classes needed?
class="img-thumbnail img-responsive"

Reply:
No change its bootstrap3 
.img-thumbnail = to give an image a rounded 1px border appearance.
.img-responsive = use for different device like mobile